ui/components/app/confirm-data/README.mdx

```diff
@@ -0,0 +1,15 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { ConfirmData } from '.';
+
+# Confirm Data
+
+Confirm Data is used on confirmation screen to display transaction data.
+
+<Canvas>
+  <Story id="components-app-ConfirmData--default-story" />
+  <Story id="components-app-ConfirmData--data-component-story" />
```

Comment:
Nice work on the README.mdx. On the design system side we use this to add comprehensive docs about our design system components but I think for app specific components just a storybook detailing all the different states and options is probably sufficient :) Sorry I should have specified that in my comments. For this to work I think you need to remove the second Story tag

```suggestion
  <Story id="components-app-ConfirmData--default-story" />
```


<img width="1440" alt="Screenshot 2023-03-06 at 1 17 27 PM" src="https://user-images.githubusercontent.com/8112138/223232907-d6a39f7b-cac7-47c3-8b11-ec8fdd21e263.png">